### PR TITLE
Added log-maxsize, one of the audit-log options avaiable for kubeadm …

### PR DESCRIPTION
**Pull request by thdrnsdk**
Until you use dynamic audit webhook, the options values for maxsize may be needed.

**What this PR does / why we need it**:
As with maxage, you must be able to set the maxsize option at install time. The Defualt value is given as 100M, which will change to another value if desired.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #67073

**Special notes for your reviewer**:
@neolit123 

**Release note**:

```release-note
kubeadm: Add argument about log size(--audit-log-maxsize). The maximum size in megabytes of the audit log file before it gets rotated
```

### CODE REVIEW

File: cmd/kubeadm/app/apis/kubeadm/types.go [hunk 1]
```diff
@@ -370,6 +370,8 @@ type AuditPolicyConfiguration struct {
 	LogDir string
 	// LogMaxAge is the number of days logs will be stored for. 0 indicates forever.
 	LogMaxAge *int32
+	// LogMaxSize is the number of Size logs will be stored for.
```
Comment (xiangpengzhao) [inline cmd/kubeadm/app/apis/kubeadm/types.go]: s/Size/size

File: cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go [hunk 2]
```diff
@@ -331,5 +331,7 @@ type AuditPolicyConfiguration struct {
 	LogDir string `json:"logDir"`
 	// LogMaxAge is the number of days logs will be stored for. 0 indicates forever.
 	LogMaxAge *int32 `json:"logMaxAge,omitempty"`
+	// LogMaxSize is the number of Size logs will be stored for.
```
Comment (xiangpengzhao) [inline cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go]: s/Size/size

**Review by xiangpengzhao — COMMENTED** [review]
just two nits. otherwise lgtm.

**Comment by thdrnsdk** [comment]
@xiangpengzhao 
changed. thanks for fast reviewed

**Comment by neolit123** [comment]
@thdrnsdk 
thank you, the change is LGTM. :+1: 
but k8s is in code freeze and we can't take it for 1.12 as it modifies a config field.
code freeze will be lifted towards the end of September.

/hold

**Comment by chuckha** [comment]
AuditPolicyConfiguration has been removed from kubeadm.

https://github.com/kubernetes/kubernetes/commit/064f74b2e8334ad0bf8550afebc407850b5044e6#diff-a6e7ba50788e0e5791c8204fcd54b00d

This can be achieved instead by using the extra-args configuration for the api-server instead.

/close

**Pull request closed without merging**